operator/internal/central/extensions/reconcile_tls_test.go

```diff
@@ -47,11 +48,11 @@ func verifySecuredClusterServiceCert(serviceType storage.ServiceType) secretVeri
 }
 
 func verifyServiceCert(serviceType storage.ServiceType, fileNamePrefix string) secretVerifyFunc {
-	return func(t *testing.T, data types.SecretDataMap) {
+	return func(t *testing.T, data types.SecretDataMap, atTime *time.Time) {
 		validatingCA, err := mtls.LoadCAForValidation(data["ca.pem"])
 		require.NoError(t, err)
 
-		assert.NoError(t, certgen.VerifyServiceCertAndKey(data, fileNamePrefix, validatingCA, serviceType))
+		assert.NoError(t, certgen.VerifyServiceCertAndKey(data, fileNamePrefix, validatingCA, serviceType, atTime))
```

Comment:
same nit as above, something along these lines:
```suggestion
		err = certgen.VerifyServiceCertAndKey(data, fileNamePrefix, validatingCA, serviceType, atTime)
		assert.NoError(t, err, "verification of certificate for service %v failed", serviceType)
```